src/lib.rs

```diff
@@ -39,7 +39,10 @@
 
 use bitflags::bitflags;
 use byteorder::{BigEndian, ByteOrder};
-use embedded_hal::i2c;
+#[cfg(not(feature = "async"))]
+use embedded_hal::i2c::I2c;
+#[cfg(feature = "async")]
+use embedded_hal_async::i2c::I2c as AsyncI2c;
```

Comment:
If you drop the rename for `AsyncI2c`, can the `maybe_async_cfg` block below get simpler?

Reply:
yup seems so!